benchmark/benchmark17_test.go

```diff
@@ -0,0 +1,64 @@
+// +build go1.7
+
+package benchmark
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/benchmark/stats"
+)
+
+func BenchClient(b *testing.B, maxConcurrentCalls, reqSize, respSize int, unaryMode, enableTrace bool) {
+	if enableTrace {
+		grpc.EnableTracing = true
+	} else {
+		grpc.EnableTracing = false
+	}
+	if unaryMode {
```

Comment:
+1.  Different workloads will have different parameters that are interesting.

OTOH, there are some parameters that will apply to all workloads (the network configuration, tracing vs. no-tracing, etc).  We should separate the two different types of parameters and group the combined ones but split out the different workloads with their parameters.

In this case, streaming and unary are separate workloads, so their parameters should be separated.  And grpc.EnableTracing should be set by the testing infrastructure separately from executing the workloads.